gui/src/components/Backups.vue

```diff
@@ -0,0 +1,130 @@
+<template>
+    <div>
+        <h4 class="pt-3">Backups in Cluster: {{cluster.cluster_name}}</h4>
+        <ClusterSummary :cluster="cluster"/>
+        <div class="row justify-content-end mb-3 pr-3">
+            <button class="btn btn-primary mx-1" data-toggle="modal" data-target="#registerBackup">Create Backup
+            </button>
+            <button
+                    type="button"
+                    class="btn btn-primary mx-1"
+                    @click="viewRestoreHistory()"
+            >View Restores
+            </button>
+        </div>
+        <table class="table text-center table-bordered">
+            <thead class="table-success">
+            <tr>
+                <th scope="col">Snapshot ID</th>
+                <th scope="col">Backup Type</th>
+                <th scope="col">Target IP</th>
+                <th scope="col">Created At</th>
+                <th scope="col">Updated At</th>
+                <th scope="col">Status</th>
+                <th scope="col"></th>
+            </tr>
+            </thead>
+            <tbody :class="{'space-groups': i !== 0 }" v-for="(b, i) in displayedBackups" :key="i">
+            <tr v-for="(e, j) in b" :key="j">
+                <td>{{e.snapshot_id}}</td>
+                <td>{{backupType(e.backup_type)}}</td>
+                <td>{{e.target_ip}}</td>
+                <th class="font-weight-normal">{{ parseInt(e.created_at) | moment('YYYY/M/D, h:mm a') }}</th>
+                <th class="font-weight-normal">{{ parseInt(e.updated_at) | moment('YYYY/M/D, h:mm a') }}</th>
+                <td>{{e.status}}</td>
+                <td>
+                    <button
+                            type="button"
+                            class="btn btn-primary mx-1"
+                            :disabled="isNotCompleted(e)"
+                            data-toggle="modal"
+                            data-target="#restoreCluster"
+                            @click="$emit('emitRestoreParams', {snapshot_id: e.snapshot_id, backup_type: e.backup_type})"
+                    >Restore
```

Comment:
I tried a restore, but the `Target IP` of the small window says `all nodes`. I think it should restore it to the IP it took backup from. (so `127.0.0.1` in this case)

![Screen Shot 2020-06-30 at 9 17 35](https://user-images.githubusercontent.com/1052473/86068572-adbcd480-bab2-11ea-8ae1-038d144e6103.png)

Reply:
I think functionality-wise, the restore behaves as intended (restores to 127.0.0.1). The return values for `Target IP(s)` in the confirmation screen is based on whether or not the request body for restore contains a `target_ips` argument or not. If no, it restores to all nodes that pertain to the backup as in the command:
```
grpcurl -plaintext -d '{"snapshot_id": "SNAPSHOT-ID", "restore_type": 2}' 192.168.0.254:20051 rpc.Cassy.RestoreBackup
```

Reply:
Then I think it is weird to restore to all the nodes if you click one backup of a node.
For example, clicking `Restore` of the backup of "192.168.0.2" in a three-node C* cluster (.2, .3, .4), will it restore to all the nodes ?

Reply:
If you don't select a `target_ip` in the Create Backup screen, it means that it will send a request to the Cassy backend with no argument for `target_ip`. So in a one-node cluster like the one in the screenshot, it will restore to "all nodes", but the only node to restore to is `127.0.0.1`.

In a three-node cluster, if you click `Restore` without specifying a `target_ip`, it will restore to all three nodes. To restore only to `192.168.0.2`, you would need to specify that on the previous screen.

Reply:
I understand, but in the current GUI, you will specify a snapshot of a particular target IP by clicking `Restore` of the target IP so I think it should always restore the specified node.  At least for the current GUI, I don't think we need an option to restore to all nodes since it is hard to specify it in the current UI.

Reply:
Ok. So for now, I will change it to automatically set the target_ip for each entry.